cli_composer.go

```diff
@@ -104,6 +104,7 @@ func composeMessageHeader(replyMsg *fbb.Message) *fbb.Message {
 
 func composeMessage(args []string) {
 	set := pflag.NewFlagSet("compose", pflag.ExitOnError)
+	from := set.StringP("from", "r", "", "")
 	subject := set.StringP("subject", "s", "", "")
```

Comment:
How about setting the default value to fOptions.Mycall here? It would simplify the if / else block below 🙂

Reply:
I can do that. I figured the code is more explicit this way, but happy to go that route. (I also looked for the ternary operator, but that's not a golang thing).